src/libPMacc/pmaccConfig.cmake

```diff
@@ -0,0 +1,167 @@
+# Copyright 2015 Erik Zenker, Rene Widera, Axel Huebl
+#
+# This file is part of libPMacc.
+#
+# libPMacc is free software: you can redistribute it and/or modify
+# it under the terms of either the GNU General Public License or
+# the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# libPMacc is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License and the GNU Lesser General Public License
+# for more details.
+#
+# You should have received a copy of the GNU General Public License
+# and the GNU Lesser General Public License along with libPMacc.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# - Config file for the pmacc package
+# It defines the following variables
+#  pmacc_INCLUDE_DIRS - include directories for pmacc
+#  pmacc_LIBRARIES    - libraries to link against
+#  pmacc_DEFINITIONS  - definitions of pmacc
+
+###############################################################################
+# pmacc
+###############################################################################
+cmake_minimum_required(VERSION 3.3)
+project("pmacc")
+
+set(pmacc_INCLUDE_DIRS ${pmacc_INCLUDE_DIRS} "${pmacc_DIR}/include")
+
+# This path should be within pmacc
+# own modules for find_packages
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
+    ${pmacc_DIR}/../../thirdParty/cmake-modules/)
+
+# Options
+option(PMACC_BLOCKING_KERNEL "activate checks for every kernel call and synch after every kernel call" OFF)
+if(PMACC_BLOCKING_KERNEL)
+  set(pmacc_DEFINITIONS ${pmacc_DEFINITIONS} "-DPMACC_SYNC_KERNEL=1")
+endif(PMACC_BLOCKING_KERNEL)
+
+set(PMACC_VERBOSE "0" CACHE STRING "set verbose level for libPMacc")
+set(pmacc_DEFINITIONS ${pmacc_DEFINITIONS} "-DPMACC_VERBOSE_LVL=${PMACC_VERBOSE}")
```

Comment:
because the project was also written in small letters. If the project name would be "PMacc" then I would call it PMacc_DEFINITIONS.